src/otx/algo/instance_segmentation/heads/transformer_decoder/maskdino_decoder.py

```diff
@@ -0,0 +1,422 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""MaskDINODecoder module."""
+from __future__ import annotations
+
+import torch
+from torch import Tensor, nn
+
+from otx.algo.detection.utils.utils import inverse_sigmoid
```

Comment:
It's werid that using module in `otx.algo.detectoin`. Could you move it to common space?

Reply:
Updated